### ISSUE TITLE
process spawning on MacOS

### ISSUE THREAD

**Issue by AshtonSBradley** [JuliaLang/PackageCompiler.jl#82]
I am not sure if this is expected behaviour or not, but once I run PackageCompiler (in particular compiling Makie, but not sure it is a Makie issue), julia keeps spawning new julia processes and throwing this warning:

```julia
julia> using IJulia
INFO: Recompiling stale cache file /Users/abradley/.julia/lib/v0.6/IJulia.ji for module IJulia.
WARNING: redefining constant JULIA_HOME
WARNING: redefining constant JULIA_HOME
WARNING: redefining constant JULIA_HOME
```
the processes sometimes close themselves, but often they stay open and/or appear to freeze.

```julia
julia> versioninfo()
Julia Version 0.6.3
Commit d55cadc350 (2018-05-28 20:20 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i5-5250U CPU @ 1.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, broadwell)
```

**Comment by KristofferC** [comment]
This package has been significantly changed and it is unclear if this is still an issue. I have precompiled Makie with no problems so it might be fixed. Please comment if otherwise.
